test/resources/visual_recognition.prepop/source_pictures.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#  Read all *.txt files in this directory, and for each create a corresponding
+#  *.zip file containing the downloaded pictures that were listed in the *.txt
+#  file.  The *.txt files included with the distribution were identified as
+#  Creative Commons licensed files acceptable for commercial reuse by Google
+#  image search.
+
+for i in *.txt; do
+  class=$(basename "$i" .txt)
+  echo "Processing ${class}..."
+  cat ${class}.txt | while read url; do
+    test -d images/$class || mkdir -p images/$class
+    (cd images/$class || exit
+    curl -O $url)
```

Comment:
Nice! Ability to recreate the .zips without including all of the image fines. I like it!